apps/src/lib/levelbuilder/rubrics/RubricsContainer.jsx

```diff
@@ -76,13 +128,31 @@ export default function RubricsContainer({
     setLearningGoalList(updatedLearningGoalList);
   };
 
-  const updateLearningGoal = (idToUpdate, keyToUpdate, newValue) => {
+  const updateLearningGoal = (
+    idToUpdate,
+    keyToUpdate,
+    newValue,
+    evidenceLevel,
+    descriptionType
```

Comment:
nit: I'm not sure what this param name means? If it the field to update?

Reply:
Fair... this parameter should either be `teacherDescription` or `aiPrompt`.   Perhaps `evidenceLevelDescriptionField` is an improvement from `descriptionType`.  But still not perfect...

Reply:
Maybe something like `evidenceLevelKeyToUpdate`?